src/ch02/trafficLight.test.ts

```diff
@@ -1,25 +1,9 @@
-import { TrafficLight } from './TrafficLight.enum';
-
-class Car {
-  stop(): boolean {
-    return false;
-  }
-
-  drive(): boolean {
-    return true;
-  }
-}
-
-const CYCLE = [TrafficLight.RED, TrafficLight.YELLOW, TrafficLight.GREEN];
+import { TrafficLight, Red, Yellow, Green } from './TrafficLight.enum';
 
+const CYCLE = [new Red(), new Green(), new Yellow()];
 const updateCarForLight = (current: TrafficLight) => {
```

Comment:
```suggestion
const updateCarForLight = (current: TrafficLight) => {
   return current.updateCar();
}
```
매개 변수는 클래스와 함수가 아닌 인터페이스로 매개변수를 사용하고 있습니다. 왜 인터페이스를 사용 했을까요? 이 부분을 찾아보고 공부를 해보면 어떨까요?

Reply:
이에 대해 공부해보니 매개변수로 사용하는 것은 그 인터페이스를 따르는 객체들을 매개변수로 할당할 수 있게 해주고
객체들은 인터페이스에 있는 함수를 상속받아 직접 구현해야 하므로 객체마다 .updateCar()의 동작이 다른 경우에도 함수에서 모두 처리할 수 있다는 것을 알았습니다.